compiler/rustc_const_eval/src/const_eval/machine.rs

```diff
@@ -291,7 +367,11 @@ impl<'mir, 'tcx> interpret::Machine<'mir, 'tcx> for CompileTimeInterpreter<'mir,
                 }
             }
 
-            if let Some(new_instance) = ecx.hook_special_const_fn(instance, args)? {
+            let Some(new_instance) = ecx.hook_special_const_fn(instance, _abi, args, dest, ret)? else {
```

Comment:
Looks like `_abi` is actually used and should be called `abi`.

Reply:
That wasn't actually used and it's gone now.